src/stdlib.cpp

```diff
@@ -141,4 +145,68 @@ auto next(const CallContext& ctx) -> Vallist {
             "bad argument #1 to 'next' (table expected, got " + a.type() + ")");
     }
 }
+
+auto select(const CallContext& ctx) -> Vallist {
+    auto index = ctx.arguments().get(0);
+    std::vector<Value> args;
+
+    for (auto a = ++ctx.arguments().begin(); a != ctx.arguments().end(); a++) {
+        args.push_back(*a);
+    }
+
+    return std::visit(
+        overloaded{
+            [&args](Number n) -> Vallist {
+                if (n == -1) {
```

Comment:
According to the lua docs negative numbers all work (not just -1) and they act as indices but starting at the end instead of the beginning. So -1 is the last element and -2 is the second last element, etc.